activemodel/test/cases/translation_test.rb

```diff
@@ -126,4 +126,14 @@ def test_human_attribute_name_does_not_modify_options
     Person.human_attribute_name("gender", options)
     assert_equal({ default: "Cool gender" }, options)
   end
+
+  require "active_support/core_ext/object/with" # at top of file
```

Comment:
Sorry I meant that this require should be at the top of this file.

Reply:
I committed the change without a close look :sweat_smile:  I'll fix it.